Detectors/TPC/reconstruction/CMakeLists.txt

```diff
@@ -22,11 +23,24 @@ set(HEADERS
    include/${MODULE_NAME}/RawReader.h
    include/${MODULE_NAME}/SyncPatternMonitor.h
    include/${MODULE_NAME}/TrackTPC.h
+   include/${MODULE_NAME}/TPCCATracking.h
 )
 Set(LINKDEF src/TPCReconstructionLinkDef.h)
 Set(LIBRARY_NAME ${MODULE_NAME})
 set(BUCKET_NAME tpc_reconstruction_bucket)
 
+if(DEFINED O2_TPCCA_TRACKING_LIB_DIR)
+  include_directories(${O2_TPCCA_TRACKING_LIB_DIR}/include)
+  FIND_LIBRARY(CATRACKING_LIB libO2TPCCATracking.so libO2TPCCATracking.dylib ${O2_TPCCA_TRACKING_LIB_DIR}/lib)
```

Comment:
Do not use `lib` prefix and extensions in `find_library`. Also, please spell `find_library` in lowercase (I also see `Set` instead of `set` above).

Please note that `find_library` will look everywhere, not only under the provided path, unless you also pass the `NO_DEFAULT_PATH` option.

The correct line would be:

```
find_library(CATRACKING_LIB "O2TPCCATracking" NO_DEFAULT_PATH)
```